test/lint/lint-locale-dependence.sh

```diff
@@ -20,7 +20,7 @@ KNOWN_VIOLATIONS=(
     "src/util/system.cpp:.*atoi"
 )
 
-REGEXP_IGNORE_EXTERNAL_DEPENDENCIES="^src/(crypto/ctaes/|leveldb/|secp256k1/|tinyformat.h|univalue/)"
+REGEXP_IGNORE_EXTERNAL_DEPENDENCIES="^src/(crypto/ctaes/|leveldb/|secp256k1/|tinyformat.h|univalue/|envvar.cpp)"
```

Comment:
This is the wrong place to put the ignore, it isn't an external dependency it is a known violation (above)

Reply:
Fixed see 912854784b11aadb907287a78748c3f75460c682